src/app/icd/client/DefaultICDClientStorage.cpp

```diff
@@ -161,7 +161,13 @@ void DefaultICDClientStorage::ICDClientInfoIteratorImpl::Release()
 CHIP_ERROR DefaultICDClientStorage::Init(PersistentStorageDelegate * clientInfoStore, Crypto::SymmetricKeystore * keyStore)
 {
     VerifyOrReturnError(clientInfoStore != nullptr && keyStore != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
+    if (mpClientInfoStore != nullptr && mpKeyStore != nullptr)
```

Comment:
nit: Maybe use `VerifyOrReturnValue` to follow the convention of params checking.

Reply:
prefer to keep the current style for multiple check here for readability consideration.